### PR TITLE
Remove Pablo Manrubia

### PR DESCRIPTION
**Pull request by AgaDufrat**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -143,7 +143,6 @@ Nick Colley <nick.colley@digital.cabinet-office.gov.uk> <nickcolley7@gmail.com>
 Nick Colley <nick.colley@digital.cabinet-office.gov.uk> <nickcolley@gds0257.local>
 Nick Stenning <nick.stenning@digital.cabinet-office.gov.uk> <nick@whiteink.com>
 Oliver Byford <oliver.byford@digital.cabinet-office.gov.uk> <o@byford.org>
-Pablo Manrubia <pablo.manrubia@digital.cabinet-office.gov.uk> <pmanrubia@gmail.com>
```
Comment (cbaines) [inline .mailmap]: Why remove Pablo from the mailmap? It's still relevant, as we haven't purged all of Pablo's commits from the repository?

Comment (AgaDufrat) [reply .mailmap]: I've seen this being done before https://github.com/alphagov/govuk-puppet/commit/54e3b6b13aae5f2016157a3ee7e33a766f04689e#diff-c9d540715cff3469b65ddd01f614848b
and I wasn't sure about keeping someone’s personal e-mail address in this file after they left.

Comment (cbaines) [reply .mailmap]: This entry is here as `pmanrubia@gmail.com` is a value used in some of the commits. Keeping it here is useful, as it means the history can be viewed in a more consistent way. Removing it won't removing `pmanrubia@gmail.com` from the history of the repository.

Comment (AgaDufrat) [reply .mailmap]: ok, will add it back

Comment (cbaines) [reply .mailmap]: Great, thanks :)

**Pull request merged**